corehq/apps/users/migrations/0072_remove_location_fields_in_user_data.py

```diff
@@ -0,0 +1,63 @@
+from django.db import migrations
+
+from couchdbkit import ResourceNotFound
+
+from corehq.apps.users.util import user_location_data
+from corehq.apps.users.models import CommCareUser
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def remove_keys_from_data(apps, schema_editor):
+    SQLUserData = apps.get_model('users', 'SQLUserData')
+
+    for user_data in SQLUserData.objects.all():
+        data = user_data.data
+        keys_to_remove = ['commcare_location_id', 'commcare_location_ids', 'commcare_primary_case_sharing_id']
+
+        modified = False
+        for key in keys_to_remove:
+            if key in data:
+                del data[key]
+                modified = True
+
+        if modified:
+            user_data.data = data
+            user_data.save()
+
+
+def revert_keys_in_data(apps, schema_editor):
```

Comment:
We don't ever actually run reverse migrations, so I'm of the opinion that we shouldn't write them.  If there were a problem and we needed to undo this change, we'd write a new migration with this code in it and deploy that.  I'm certainly not opposed to preparing for the worst, just want to be explicit about expectations.

Reply:
https://github.com/dimagi/commcare-hq/pull/34852#issuecomment-2252993403
^ though this is a decent enough reason to keep it around

Reply:
Thank you Ethan. I always thought in general we want to write reverse functions for migration. Is there any harm to leave the reverse function in a migration file?

Reply:
nope, no harm leaving it in

Reply:
Thank you! I'll just leave it here.